src/Components/Articles/ArticlePage.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticleById } from "../../api";
+
+export const ArticlePage = () => {
+  const { articleId } = useParams();
+  const [article, setArticle] = useState();
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getArticleById(articleId)
+      .then((articleData) => {
+        setArticle(articleData);
+        setIsLoading(false);
+      })
+      .catch((error) => {
+        console.error("Error fetching article:", error);
+        setIsLoading(false);
+      });
+  }, [articleId]);
+
+  if (isLoading) {
+    return <h3>Loading...</h3>;
+  }
+
+  return (
+    <div className="ArticlePage">
+      <h1>{article.title}</h1>
+      <p>By: {article.author}</p>
+      <p>Published: {article.created_at}</p>
```

Comment:
The date here is in a bit of an unreadable format, check out the toUTCString method that may give you a hand formatting this a little nicer